lib/fastlane/helper/sh_helper.rb

```diff
@@ -35,8 +35,12 @@ def self.sh_no_action(command, log: true)
 
         if exit_status != 0
           # this will also append the output to the exception
-          message = "Exit status of command '#{command}' was #{exit_status} instead of 0."
-          message += "\n#{result}" if log
+          if log || !is_ci?
```

Comment:
We assumed that more verbose logging would still be valuable - with no significant risk - if the user was running from their local machine, but happy to be persuaded otherwise!